File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    for j in range(len(commands)):
        arr= commands[j]
        li = []
        for i in range(arr[0]-1,arr[1]):
            li.append(array[i])
            li.sort()
        answer.append(li[arr[2]-1])
    return answer


    

File: 프로그래머스/1/42748. K번째수/K번째수.js
function solution(array, commands) {
    var answer = [];
    const len = commands.length;
    //commands의 length만큼 반복한다. 
    for (let n = 0; n < len ; n++){
        let i = commands[n][0]-1
        let j = commands[n][1]-1
        let k = commands[n][2]-1
        let targetNumber = array.slice(i,j+1).sort((a,b)=>a-b)[k]
        answer.push(targetNumber)
    }
    return answer;
}